mlrun/utils.py

```diff
@@ -53,6 +53,12 @@ def create_logger(stream=None):
     return logger
 
 
+def set_logger_level(level):
+    logger.setLevel(level)
+    for handler in logger.handlers:
+        handler.setLevel(level)
```

Comment:
I think doing that on the logger is enough, why also on the handlers ?

Reply:
I see now why you did it - because in `create_logger` we're setting the handlers log level - I think that's wrong, if we'll set the handlers to the lowest log level we won't need to re-set their log level